<launch>
    <include file="$(find plane_camera_magnet)/launch/directed_assembly_cam.launch"> 
            <arg name="device" value="14434226"/>
    </include>
    <node   pkg="image_proc"
            name="image_proc"
            type="image_proc"
            ns="pg_14434226">
    </node>
    <node   pkg="image_view" 
            name = "disp"
            type = "image_view" 
            args="image:=/pg_14434226/image_raw" >
    </node>
    <node name="datrackdisc"
            pkg="plane_camera_magnet" 
            type = "datrackdisc"
            output="screen">
            <param name="cal_file" value = "$(find plane_camera_magnet)/calib/calda.yml"/>
            <param name="param_file" value = "$(find plane_camera_magnet)/calib/paramda.yml"/>
    </node>
</launch>
